kafka-bridge.go

```diff
@@ -241,6 +247,17 @@ func extractTID(msg string) (tid string, err error) {
 	return tid, nil
 }
 
+func extractOriginSystem(msg string) (string, error) {
+	origSysHeaderRegexp := regexp.MustCompile(`Origin-System-Id:\s[a-zA-Z0-9:/.-]*`)
```

Comment:
this can move to a package variable, so you are not compiling the regex every time.